### DISCUSSION TITLE
recipes/reference/temperatures/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#900] [Comments]
# recipes/reference/temperatures/

A collection of my recipes.

https://nicholaswilde.io/recipes/reference/temperatures/

**Comment by nicholaswilde** [comment]
160 F egg whites become pasteurized (swiss meringue)

**Reply by nicholaswilde** [reply]
Temperature at which bread is cooked.

**Reply by nicholaswilde** [reply]
The ideal internal temperature for cooked bread can vary depending on the type of bread you're baking. Here's a general guide:
 * Lean dough breads (e.g., sourdough, baguettes): 190°F - 210°F (88°C - 99°C)
 * Enriched dough breads (e.g., brioche, challah): 180°F - 190°F (82°C - 88°C)
Using a thermometer is the most accurate way to determine if your bread is done. In addition to temperature, you can also check for visual cues like crust color.
